### PR TITLE
feat : update readme cargo loading

### PR DESCRIPTION
**Pull request by sfukuta**
Description:
- Update documents about cargo loading service.

Info:
- update node graph.
- update node info.
- update map info.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ It is necessary to prepare a user-defined broadcasting device, which connects to
     - `/autoware_state_machine/state` \[[autoware_state_machine_msgs/msg/StateMachine](https://github.com/eve-autonomy/autoware_state_machine_msgs/blob/main/msg/StateMachine.msg)\]:<br>State of the system.
   - from user-defined broadcasting device
     - `v2i status` ([UDP](#v2i-status)) :<br>State from V2I infrastructure. It has an array structure to control the vehicle based on the state of multiple infrastructures.
+  - from [cargo_loading_service](https://github.com/eve-autonomy/cargo_loading_service)
+    - `/awapi/tmp/infrastructure_commands` \[[tier4_v2x_msgs/msg/InfrastructureCommandArray](https://github.com/tier4/tier4_autoware_msgs/blob/tier4/universe/tier4_v2x_msgs/msg/InfrastructureCommandArray.msg)\]:<br>Control command to V2I infrastructure. It has an array structure to control multiple infrastructures at the same time.
```
Comment (yn-mrse) [inline README.md:19]: Inputs of the same system can be easily understood by the user by describing them consecutively.
Please move between L.13 and L.14.

Comment (yn-mrse) [inline README.md:19]: The topic name is the same as autoware, but is it wrong?
```suggestion
  - from [cargo_loading_service](https://github.com/eve-autonomy/cargo_loading_service)
    - `/cargo_loding/infurastructre_commands` \[[tier4_v2x_msgs/msg/InfrastructureCommandArray](https://github.com/tier4/tier4_autoware_msgs/blob/tier4/universe/tier4_v2x_msgs/msg/InfrastructureCommandArray.msg)\]:<br>Control command to V2I infrastructure. It has an array structure to control multiple infrastructures at the same time.
```

File: README.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ It is necessary to prepare a user-defined broadcasting device, which connects to
...
 - output
```
Comment (yn-mrse) [inline README.md:20]: It looks like there are more outputs for cargo_loading_service in the node graph, so please add a corresponding topic.

**Review by yn-mrse — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sfukuta) [reply README.md]: upda https://github.com/eve-autonomy/v2i_interface/pull/39/commits/92560c5be4b9910a50f968396e6d13beea742d9f

File: README.md (hunk 2 above)
Comment (sfukuta) [reply README.md]: update https://github.com/eve-autonomy/v2i_interface/commit/92560c5be4b9910a50f968396e6d13beea742d9f

File: README.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ It is necessary to prepare a user-defined broadcasting device, which connects to
 - input
   - from [autoware.universe](https://github.com/autowarefoundation/autoware.universe/)
     - `/awapi/tmp/infrastructure_commands` \[[tier4_v2x_msgs/msg/InfrastructureCommandArray](https://github.com/tier4/tier4_autoware_msgs/blob/tier4/universe/tier4_v2x_msgs/msg/InfrastructureCommandArray.msg)\]:<br>Control command to V2I infrastructure. It has an array structure to control multiple infrastructures at the same time.
+    - from [cargo_loading_service](https://github.com/eve-autonomy/cargo_loading_service)
```
Comment (yn-mrse) [inline README.md:14]:
```suggestion
  - from [cargo_loading_service](https://github.com/eve-autonomy/cargo_loading_service)
```

File: docs/node_graph.pu [hunk 4]
```diff
@@ -14,12 +16,15 @@ rectangle user-defined {
...
 
-(/awapi/awapi_awiv_adapter_node) --> (v2i_interface) : /awapi/tmp/infrastructure_commands
-(/awapi/awapi_awiv_adapter_node) <-- (v2i_interface) : /system/v2x/virtual_traffic_light_states
-(autoware_state_machine) --> (v2i_interface) : /autoware_state_machine/state
-(v2i_interface) --> (broadcasting device) : v2i command (UDP)
-(v2i_interface) <-- (broadcasting device) : v2i status (UDP)
+(/awapi/awapi_awiv_adapter_node) --> (v2i_interface_module) : /awapi/tmp/infrastructure_commands
+(/awapi/awapi_awiv_adapter_node) <-- (v2i_interface_module) : /system/v2x/virtual_traffic_light_states
+(autoware_state_machine) --> (v2i_interface_module) : /autoware_state_machine/state
+(cargo_loading_service) ---> (v2i_interface_module) : /cargo_loding/infurastructre_commands
```
Comment (yn-mrse) [inline docs/node_graph.pu:24]:
```suggestion
(cargo_loading_service) ---> (v2i_interface_module) : /cargo_loding/infrastructure_commands
```

File: docs/node_graph.pu [hunk 5]
```diff
@@ -14,12 +16,15 @@ rectangle user-defined {
...
+usecase "cargo_loading_service"
 
-(/awapi/awapi_awiv_adapter_node) --> (v2i_interface) : /awapi/tmp/infrastructure_commands
-(/awapi/awapi_awiv_adapter_node) <-- (v2i_interface) : /system/v2x/virtual_traffic_light_states
-(autoware_state_machine) --> (v2i_interface) : /autoware_state_machine/state
-(v2i_interface) --> (broadcasting device) : v2i command (UDP)
-(v2i_interface) <-- (broadcasting device) : v2i status (UDP)
+(/awapi/awapi_awiv_adapter_node) --> (v2i_interface_module) : /awapi/tmp/infrastructure_commands
+(/awapi/awapi_awiv_adapter_node) <-- (v2i_interface_module) : /system/v2x/virtual_traffic_light_states
+(autoware_state_machine) --> (v2i_interface_module) : /autoware_state_machine/state
```
Comment (yn-mrse) [inline docs/node_graph.pu:23]:
```suggestion
(autoware_state_machine) -> (v2i_interface_module) : /autoware_state_machine/state
```
Suggestion for layout modification.

**Review by yn-mrse — CHANGES_REQUESTED** [review]

File: docs/node_graph_internal_spec.pu [hunk 6]
```diff
@@ -0,0 +1,53 @@
+@startuml
+
+rectangle autoware {
+  usecase "/awapi/awapi_awiv_adapter_node"
```
Comment (yn-mrse) [inline docs/node_graph_internal_spec.pu:4]:
```suggestion
  interface "/awapi/tmp/infrastructure_commands"
  interface "/system/v2x/virtual_traffic_light_states"
```

File: docs/node_graph_internal_spec.pu [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+}
+usecase "autoware_state_machine"
```
Comment (yn-mrse) [inline docs/node_graph_internal_spec.pu:6]:
```suggestion
interface "/autoware_state_machine/state"
```

File: docs/node_graph_internal_spec.pu [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+rectangle "v2i_interface" as v2i {
+usecase "v2i_interface" #LightCoral
+usecase "v2i_command_muxer" #LightCoral
+usecase "vtl_adapter" #LightCoral
```
Comment (yn-mrse) [inline docs/node_graph_internal_spec.pu:10]:
```suggestion
  usecase "v2i_interface" #LightCoral
  usecase "v2i_command_muxer" #LightCoral
  usecase "vtl_adapter" #LightCoral
```

File: docs/node_graph_internal_spec.pu [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+} 
+rectangle user-defined {
+  node "broadcasting device"
+  node "v2i controller[0]" as v2ictl_0
+  node "v2i controller[1]" as v2ictl_1
+  node "v2i controller[n]" as v2ictl_n
+  node "v2i infrastructure[0]" as v2ihw_0
+  node "v2i infrastructure[1]" as v2ihw_1
+  node "v2i infrastructure[n]" as v2ihw_n
+}
```
Comment (yn-mrse) [inline docs/node_graph_internal_spec.pu:20]:
```suggestion
interface "v2i command (UDP)" as v2i_command_udp
interface "v2i status (UDP)" as v2i_status_udp
```

File: docs/node_graph_internal_spec.pu [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+usecase "cargo_loading_service"
```
Comment (yn-mrse) [inline docs/node_graph_internal_spec.pu:21]:
```suggestion
interface "/cargo_loding/infurastructre_commands"
```

File: docs/node_graph_internal_spec.pu [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+
+(/awapi/awapi_awiv_adapter_node) --> (vtl_adapter) : /awapi/tmp/infrastructure_commands
+(/awapi/awapi_awiv_adapter_node) <-- (vtl_adapter) : /system/v2x/virtual_traffic_light_states
+(autoware_state_machine) --> (vtl_adapter) : /autoware_state_machine/state
```
Comment (yn-mrse) [inline docs/node_graph_internal_spec.pu:25]:
```suggestion
(/awapi/tmp/infrastructure_commands) --> (vtl_adapter)
(/system/v2x/virtual_traffic_light_states) <-- (vtl_adapter)
(/autoware_state_machine/state) --> (vtl_adapter)
```

File: docs/node_graph_internal_spec.pu [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+(cargo_loading_service) ---> (v2i_command_muxer) : /cargo_loding/infurastructre_commands
```
Comment (yn-mrse) [inline docs/node_graph_internal_spec.pu:26]:
```suggestion
(/cargo_loding/infurastructre_commands) --> (v2i_command_muxer)
```

File: docs/node_graph_internal_spec.pu [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+(vtl_adapter) --> (v2i_command_muxer) : /v2i_gate/infrastructure_commands
+(v2i_command_muxer) --> (v2i_interface) : /v2i/infrastructure_commands
+(vtl_adapter) <-- (v2i_interface) :/v2i/infrastructer_states
+(cargo_loading_service) <-- (v2i_interface) :/v2i/infrastructer_states
```
Comment (yn-mrse) [inline docs/node_graph_internal_spec.pu:30]:
```suggestion
```

File: docs/node_graph_internal_spec.pu [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+(v2i_interface) --> (broadcasting device) : v2i command (UDP)
+(v2i_interface) <-- (broadcasting device) : v2i status (UDP)
```
Comment (yn-mrse) [inline docs/node_graph_internal_spec.pu:32]:
```suggestion
(v2i_interface) --> (v2i_command_udp)
(v2i_interface) <-- (v2i_status_udp)
```

File: docs/node_graph_internal_spec.pu [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+(v2ictl_0) --> (v2ihw_0)
+(v2ictl_0) <-- (v2ihw_0)
+(v2ictl_1) --> (v2ihw_1)
+(v2ictl_1) <-- (v2ihw_1)
+(v2ictl_n) --> (v2ihw_n)
+(v2ictl_n) <-- (v2ihw_n) : GPIO communication
+
+note right of "broadcasting device" : In vehicle
+note right of v2ictl_n : On infrastructure ( off-vehicle )
+
```
Comment (yn-mrse) [inline docs/node_graph_internal_spec.pu:52]:
```suggestion
```

**Review by yn-mrse — CHANGES_REQUESTED** [review]
Since node_graph_internal_spec.pu is an internal specification, there is no need to specify external components.
image

**Comment by sfukuta** [comment]
@yn-mrse 
Fixed an issue that resulted from a suggestion to the node graph internal spec.

File: README.md (hunk 2 above)
Comment (sfukuta) [reply README.md:20]: update https://github.com/eve-autonomy/v2i_interface/pull/39/commits/c120cc89c5b1e291a13483992f312f134a5fc7cf

File: README.md [hunk 16]
```diff
@@ -11,35 +11,25 @@ It is necessary to prepare a user-defined broadcasting device, which connects to
...
   - to [autoware.universe](https://github.com/autowarefoundation/autoware.universe/)
     - `/system/v2x/virtual_traffic_light_status` \[[tier4_v2x_msgs/msg/VirtualTrafficLightStateArray](https://github.com/tier4/tier4_autoware_msgs/blob/tier4/universe/tier4_v2x_msgs/msg/VirtualTrafficLightStateArray.msg)\]:<br>ROS2 interface from `v2i_status` (UDP).
+  - to [cargo_loading_service](https://github.com/eve-autonomy/cargo_loading_service)
+    - `/v2i/infrastructer_states` \[[v2i_interface_msgs/msg/InfrastructureStateArray.msg](https://github.com/eve-autonomy/v2i_interface_msgs/blob/main/msg/InfrastructureState.msg)\]:<br>ROS2 interface from `v2i_status` (UDP).
   - to user-defined broadcasting device
     - `v2i command` ([UDP](#v2i-command)) :<br>UDP protocol from `/awapi/tmp/infrastructure_commands`.
 
 ## Node Graph
+### v2i interface ｍodule external specifications
 ![node graph](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/eve-autonomy/v2i_interface/main/docs/node_graph.pu)
```
Comment (yn-mrse) [inline README.md:30]:
```suggestion
![node graph](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/eve-autonomy/v2i_interface/main/docs/node_graph_external_spec.pu)
```

**Review by yn-mrse — CHANGES_REQUESTED** [review]

File: README.md (hunk 16 above)
Comment (yn-mrse) [reply README.md]: I've confirmed that it's fixed in https://github.com/eve-autonomy/v2i_interface/pull/39/commits/ecc4a47ca0edcf036b816b130c64cbe0ac959e89 .

**Review by yn-mrse — APPROVED** [review]

**Pull request merged**